javascript/javascript1/week2/candy-helper.js

```diff
@@ -0,0 +1,46 @@
+const myCandyBasket = [{ candyType: "sweet", pricePerGram: 0.5 }, { candyType: "toffee", pricePerGram: 1.1 },
+{ candyType: "Chocolate", pricePerGram: 0.7 }, { candyType: "chewingGum", pricePerGram: 0.03 }];
+const boughtCandyPrices = [];
+let amountToSpend = Math.random() * 100;
+let flag = true;
+console.log(`Total money in hand is ${amountToSpend}`);
+console.log(`My candy basket with there type and respective price  pergram is`);
+console.log(myCandyBasket);
+
+function addCandy(name, weight) {
```

Comment:
according to the exercise this `addCandy` function has to calculate and add the price to the `boughtCandyPrices`. but you used another function which creates confusion.

Reply:
I understand and will do,thank you.